manly.py

```diff
@@ -82,7 +81,7 @@ def manly(program, flags=''):
     # we set MANWIDTH, so we don't rely on the users terminal width
     # try `export MANWIDTH=80` -- makes manuals more readable imo :)
     process = subprocess.Popen(
-        "export MANWIDTH=80; man %s" % shlex.quote(program),
```

Comment:
We should quote it.

Here is `shlex.quote` from the 3.8 stdlib:
``` python
_find_unsafe = re.compile(r'[^\w@%+=:,./-]', re.ASCII).search

def quote(s):
    """Return a shell-escaped version of the string *s*."""
    if not s:
        return "''"
    if _find_unsafe(s) is None:
        return s

    # use single quotes, and put single quotes into double quotes
    # the string $'b is then quoted as '$'"'"'b'
    return "'" + s.replace("'", "'\"'\"'") + "'"
```

Alternatively, we can put a do-nothing function in on `ImportError`. What do you think?